test/plots/barPlotTests.ts

```diff
@@ -506,6 +506,38 @@ describe("Plots", () => {
         svg.remove();
       });
 
+      it("hides labels properly on the right", () => {
```

Comment:
Let's maybe extract these in their own describe. It is a bit hard to follow the test.